<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefVideo</class>
 <widget class="QWidget" name="PrefVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QCheckBox" name="postprocessing_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable postprocessing by default</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_slices_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>132</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw video using slices</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="direct_rendering_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>76</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Direct rendering</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="double_buffer_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>104</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Double buffering</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="eq2_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>48</y>
     <width>448</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Use software video equalizer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vo_combo">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>170</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="vo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Output driver:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring>vo_combo</cstring>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
